general:
  # Select location from list below
  default_location: "Custom"
  # Use map as background 0: no map, 1: osm, 2: google satellite
  use_map: 1

radar:
  # Distance in kilometers. Inner radius of the square frame.
  frame_rain: 1
  frame_alert: 9
  frame_warning: 20
  # Miniumum percent of rain in frame. Value bellow will be ignored as a noise.
  min_rain: 0
  min_alert: 1
  min_warning: 0.5
  snow: 0
  smooth: 1
  # Possible values 256 or 512 in px
  size: 512
  zoom: 10
  # Rain threshold 215 default (remove gray zone)
  rain_threshold: 250
  # Color scheme values 0-8. See https://www.rainviewer.com/api/color-schemes.html
  # For threshold recommended 1
  color: 1


# List of locations

# In custom location you can use your own lat and lon values
Custom:
  lat: 50.0755828
  lon: 14.4435161

Prague:
  lat: 50.0755828
  lon: 14.4435161

Graisbach_DE:
  lat: 48.7486492
  lon: 10.8969919

Lanaja_SP:
  lat: 41.773308
  lon: -0.326232

Dresden_DE:
  lat: 51.044417
  lon: 13.663099

Berlin_DE:
  lat: 52.316536
  lon: 13.646875

Breda_NL:
  lat: 51.546366
  lon: 4.729406

Dijon_FR:
  lat: 47.35233566273505
  lon: 4.950463858297299